semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/db/DbSemanticConventionUtils.java

```diff
@@ -106,6 +110,18 @@ public static boolean isMongoBackend(Event event) {
     return Lists.newArrayList(Sets.newHashSet(OTEL_DB_STATEMENT));
   }
 
```

Comment:
Can you change access modifier for methods which are just used internally (also can you order the methods so that public methods are at the top)?